support/modules-artifact-gen/single-file-artifact-gen

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
(I know this is pre-existing, as the rest of the scripts were already using `bash`)

When possible, we should limit ourselves to `sh` syntax which is more portable. See below for an alternative command for `<<<`